packages/rum-core/src/common/config-service.js

```diff
@@ -76,15 +76,11 @@ class Config {
       debug: false,
       logLevel: 'warn',
       breakdownMetrics: false,
-      checkBrowserResponsiveness: true,
-      groupSimilarSpans: true,
-      similarSpanThreshold: 0.05,
```

Comment:
Did we forget to remove this previously ?

Reply:
Yes 🤦‍♂just the option.